docs/python/environments.md

```diff
@@ -21,19 +21,21 @@ This command automatically looks for and displays a list of available Python int
 
 ![List of interpreters](images/environments/interpreters-list.png)
 
-Selecting an interpreter from the list configures the `settings.json` file accordingly.
+Selecting an interpreter from the list configures the `settings.json` file accordingly. The statusbar will also be updated to show the selected interpreter, which can also be clicked on to bring up the list of available Python interpreters. If no interpreter is selected, the statusbar will reflect this fact:
```

Comment:
Edit out the "will" words:  "The statusbar also shows the selected interpreter, which brings up a list of available interpreters when selected. The statusbar also reflects when no interpreter is selected."